packages/quick_actions/quick_actions_platform_interface/lib/method_channel/method_channel_quick_actions.dart

```diff
@@ -24,6 +25,7 @@ class MethodChannelQuickActions extends QuickActionsPlatform {
       assert(call.method == 'launch');
       handler(call.arguments);
     });
+    unawaited(channel.invokeMethod<String?>('initialize'));
```

Comment:
If you're going to call a new method, you should make it a no-op on iOS to avoid getting useless error messages (although per other comments, it's not clear to me that this is necessary).

Reply:
Removed as it's not necessary.